lib/components/toasts/style.less

```diff
@@ -1,6 +1,7 @@
 @import "../../variables.less";
 
 @{prefix}toast {
+  display: flex;
```

Comment:
@jkodu this is already happening [below via the `.display-flex()` mixin](https://github.com/dequelabs/pattern-library/pull/128/files#diff-764ba649d62426b192e33d3c9901c784L11)

Reply:
**NOTE**: you have to expand the visible lines in order to see it 😄

Reply:
Good catch @schne324.

I debugged and found that the `display:flex` actually happens due to the cascading style coming from `.dqpl-fadein-setup`.

I realised the actual fix is to swap `.dqpl-fadein-setup` with `.dqpl-fade-in-flex`. 

That fix is not within `pattern-library` repository. So I will close this & look into putting a fix in `cauldron-react`.

![image](https://user-images.githubusercontent.com/20978252/66769157-53d89800-eeac-11e9-9417-8f86d8814722.png)

Reply:
ahah! `dqpl-fade-setup` is the culprit - good find!